llvm/include/llvm/MC/CAS/MCCASObjectV1.def

```diff
@@ -35,6 +37,8 @@ CASV1_SIMPLE_GROUP_REF(DebugAbbrevSectionRef, mc:debug_abbrev_section)
 CASV1_SIMPLE_GROUP_REF(DebugLineSectionRef, mc:debug_line_section)
 CASV1_SIMPLE_GROUP_REF(AtomRef, mc:atom)
 CASV1_SIMPLE_GROUP_REF(SymbolTableRef, mc:symbol_table)
+CASV1_SIMPLE_GROUP_REF(DIEAbbrevSetRef, mc:debug_DIE_abbrev_set)
+CASV1_SIMPLE_GROUP_REF(DIETopLevelRef, mc:debug_DIE_top_level)
```

Comment:
Is there one DIETopLevelRef per translation unit? If so, why is it any different from a`DebugInfoSectionRef`?

Reply:
>Is there one DIETopLevelRef per translation unit? 

Yeah

> If so, why is it any different from a DebugInfoSectionRef?

This is both to ease the transition period and also to put all blocks semantically related under the same parent, so that they are easy to find and materialize. The cost is negligible.

Reply:
Sounds good to me